target/openrisc/cpu.c

```diff
@@ -51,7 +51,7 @@ static void openrisc_cpu_reset(CPUState *s)
     s->exception_index = -1;
 
     cpu->env.upr = UPR_UP | UPR_DMP | UPR_IMP | UPR_PICP | UPR_TTP;
-    cpu->env.cpucfgr = CPUCFGR_OB32S | CPUCFGR_OF32S;
+    cpu->env.cpucfgr = cpu->feature;
```

Comment:
thanks I was thinking about this change too, it was strange to have the mask set 2 times.